sqlglot/dialects/clickhouse.py

```diff
@@ -397,6 +397,7 @@ class Generator(generator.Generator):
             exp.Pivot: no_pivot_sql,
             exp.Quantile: _quantile_sql,
             exp.RegexpLike: lambda self, e: f"match({self.format_args(e.this, e.expression)})",
+            exp.Rand: rename_func("randCanonical"),
```

Comment:
don't we need to parse randCanonical -> rand?

Reply:
ah, yes